Formula/civl.rb

```diff
@@ -15,18 +16,23 @@ class Civl < Formula
     sha256 cellar: :any_skip_relocation, all: "c6c4de30805b69e53aacb7fb0069b67f42c48983418b3d23c97b9ef7013fe715"
   end
 
-  depends_on :macos # ships non-portable prebuilt Linux binaries
   depends_on "openjdk"
   depends_on "z3"
 
+  resource("testdata") do
+    url "https://vsl.cis.udel.edu/lib/sw/civl/1.20/r5259/release/CIVL-1.20_5259.tgz"
+    sha256 "15bf63b3a92300e8432e95397284e29aaa5897e405db9fc2d56cd086f9e330d3"
+  end
+
   def install
     underscored_version = version.to_s.tr("-", "_")
     libexec.install "lib/civl-#{underscored_version}.jar"
     bin.write_jar_script libexec/"civl-#{underscored_version}.jar", "civl"
-    pkgshare.install "doc", "emacs", "examples", "licenses"
```

Comment:
We can keep the doc, btw. That doesn't cause any audit failures. Unless the docs take up a large amount of space?

Reply:
It's a pdf, I added it back.

Reply:
Note: technically speaking I almost consider a pdf a binary file :D

Reply:
That is correct, but not what we mean by "pre-built binaries" 😄